tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -6162,6 +6176,11 @@ declare namespace ts {
     /** The classifier is used for syntactic highlighting in editors via the TSServer */
     function createClassifier(): Classifier;
 }
+declare namespace ts.classifier.vscode {
+    /** This is mainly used internally for testing */
```

Comment:
This shouldnt be in the api... mark internal.. Infact current namespace ts.classifier.vscode  should be internal